packages/labs/src/protobufjs/ts_proto_library.bzl

```diff
@@ -104,16 +107,29 @@ def _ts_proto_library(ctx):
     dts = _run_pbts(ctx.actions, ctx.executable, js_es5)
 
     # Return a structure that is compatible with the deps[] of a ts_library.
+    declarations = depset([dts])
+    es5_sources = depset([js_es5])
+    es6_sources = depset([js_es6])
+
     return struct(
-        files = depset([dts]),
+        providers = [
+            DefaultInfo(files = declarations),
+            DeclarationInfo(
+                declarations = declarations,
+                transitive_declarations = declarations,
+            ),
+            JSEcmaScriptModuleInfo(
+                sources = es6_sources,
+            ),
+        ],
         typescript = struct(
```

Comment:
does this do anything?

Reply:
yes. needed to update examples/protocol_buffers which ties ts_proto_library -> ts_library -> rollup_bundle